warehouse/templates/packaging/detail.html

```diff
@@ -282,6 +282,12 @@ <h3 class="sidebar-section__title">{% trans %}Navigation{% endtrans %}</h3>
 
         {# Tab: Project description #}
         <div id="description" data-project-tabs-target="content" class="vertical-tabs__content" role="tabpanel" aria-labelledby="description-tab mobile-description-tab" tabindex="-1">
+          {% if release.yanked and release.yanked_reason %}
+            <div class="callout-block callout-block--warning">
```

Comment:
Nice usage of the blocks - had you considered using the `callout-block--danger` instead ([ref](https://github.com/pypi/warehouse/blob/695a3b9549cb4cdf8226aa2d57ab7d419af25c84/warehouse/static/sass/blocks/_callout-block.scss#L36-L37))? That lines up more clearly with the badge at the top of the page.
<img width="264" alt="Screenshot 2023-12-18 at 10 51 17" src="https://github.com/pypi/warehouse/assets/529516/b5e1ff7d-18fa-49cd-bf65-529e6544b3be">

Reply:
I have no personal opinion on either the warning or danger callout, so I'll go with the danger one.